src/frontend-pwa/src/App.tsx

```diff
@@ -31,7 +31,7 @@ function App() {
     //  TODO:This will be removed once we actually use isLoading during API calls, Loading media etc
     setTimeout(() => {
       setIsLoading(false);
-    }, 5000);
+    }, 1);
```

Comment:
1 millisecond?

Reply:
hacky stub timer to avoid waiting the full five seconds. (done on my own)
after talking with the team, we decided that if we're going to set that timer that low, it may as well not exist at all. 
so, we just set setIsLoading() within the useEffect, rather than a separate timer.
that is of course not ideal either, but will be remedied in a ticket this sprint, we recognize the hackiness.

Reply:
i will leave this unresolved for now, so as not to break our comment chain in case you have any remarks!